### DISCUSSION TITLE
Invert function?

### DISCUSSION THREAD

**Discussion by jimjg** [ImageMagick/ImageMagick#5004] [Digital Image Processing] [answered]
For my simple script, I need to invert the blue channel of a png image (for OCR).  It has to be via a command line.  "Posturize" then "Invert"ing works well enough in gimp2.

**Comment by fmw42** [comment, marked as answer]
`convert image.suffix -channel blue -negate +channel result.suffix`

If on IM 7, use magick in place of convert

**Reply by jimjg** [reply]
I missed the -invert function
Thanks!
